README.md

```diff
@@ -13,26 +13,10 @@ Playwright with PyTest Test runner Integration with BrowserStack.
 * To run your automated tests using BrowserStack, you must provide a valid username and access key. This can be done by setting the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY environment variables.
```

Comment:
Modify this line to change inside browserstack yml

Reply:
cool, either one will work though